### ISSUE TITLE
Wrong Colours 

### ISSUE THREAD

**Issue by Feldmarschall** [TownyAdvanced/MapTowny#40]
I switched from Dynmap-Towny to MapTowny. Unfortunately, almost all colors are completely bugged. If I select nation color black, it is blue. If I choose dark green, it's purple and so on. Dark blue is light blue

### Plugin Version
Towny - 0.98.4.4
Dynmap - 3.4
LiveAtlas - 3.1.0
Maptowny - 2.0.1 

### Server Software Version
 Paper version git-Paper-298 (MC: 1.19.2) (Implementing API version 1.19.2-R0.1-SNAPSHOT) (Git: 309133e)

MapTowny config

```
# Worlds that should display town claims.
enabled-worlds:
- world

# How often should the plugin render all towns? (In minutes)
update-period: 5

# Layer Properties
layer:
  # Name of the layer
  name: Towns
  # Should the layer be toggleable by users?
  show-controls: true
  # Should the layer be invisible by default?
  default-hidden: false
  # Layer Priority.
  # Don't need to touch this unless other Pl3xMap add-ons are interfering with the layer.
  layer-priority: 5
  # The z-index on which the layer will display.
  # Decrease if you want the layer to be more blended in with the map.
  z-index: 250

# Fill Style.
# Properties about how claims should look on the map
fill-style:
  # Whether to fill the claim with color
  fill: true
  fill-color: '#050505'
  fill-opacity: 0.25
  # Use specified nation color as the fill color instead?
  use-nation-color-fill: true
  
  # Whether to draw a stroke along the claim path.
  stroke: true
  # Stroke width in pixels
  stroke-weight: 3
  stroke-color: '#1BFF00'
  stroke-opacity: 1.0
  # Use specified nation color as the stroke color instead?
  use-nation-color-stroke: true
  # Use specified town color as the fill color instead?
  # This option will take priority over the 'use-nation-color-fill' option if it is enabled.
  use-town-color-fill: false
  # Use specified town color as the stroke color instead?
  # This option will take priority over the 'use-nation-color-stroke' option if it is enabled.
  use-town-color-stroke: false

# Icon Properties:
# Icons are placed at the center of a town's homeblock.
icon-info:
  # Icon for the town's homeblock. Icon must be a valid image URL.
  # Default Icon created by icon king1 licensed under Creative Commons 3.0
  # https://creativecommons.org/licenses/by/3.0/
  town-icon: default
  # Icon for a town if they are the capital of the nation. Icon must be a valid image URL.
  # Put 'default' to use the town icon image.
  capital-icon: default
  # Icon for an outpost claim that will appear at the location of outpost spawns. Icon must be a valid image URL.
  # Put 'default' to use the town icon image.
  # Put 'empty' to not place icons at outposts.
  outpost-icon: empty
  # Height of the icon
  icon-height: 35
  # Width of the icon
  icon-width: 35
# Do not touch!!!
config-version: 1.0
```

Example Nation:

```
capital=Black_Castle
nationBoard=/nation set board [msg]
mapColorHexCode=000000
allies=Livera,Auridion,Atmuntaris,Domus_Aurea,Insulae,Arnor,Raurun
enemies=
taxpercent=false
maxPercentTaxAmount=10000.0
taxes=0.0
spawnCost=0.0
neutral=false
uuid=871c8754-649e-4d02-afed-cde560c8f0a0
registered=1599155884134
isPublic=false
isOpen=false
metadata=
```
![image](https://user-images.githubusercontent.com/1348940/206723321-afb07b73-a253-446f-8a5f-a667fd79380e.png)

**Comment by silverwolfg11** [comment]
Thank you for reporting this. I'll hopefully have time to take a look into this soon.

**Comment by LcyDev** [comment]
Still happening

**Comment by silverwolfg11** [comment]
Sorry for such a delayed fix, but this should be resolved in [v2.1.1](https://github.com/TownyAdvanced/MapTowny/releases/tag/v2.1.1). 

I tested this fix on the latest Dynmap (v3.5-beta-1) using Towny 0.98.3 on a local paper Server (PaperMC Build #383). If you still experience issues with this, let me know.
